docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - redis
 
   db:
-    image: library/postgres:latest
+    image: library/postgres:11.2
```

Comment:
In the tests we are using ```9.4``` https://github.com/mirumee/saleor/blob/1a2480fa0c08a6730bf00fca536b68c52f10a7ba/.circleci/config.yml#L28, maybe we could keep them same?

Reply:
Thanks, new to saleor, so didn't notice the Circle CI config also has docker images. Will check if similar thing is there for redis image & will update the PR accordingly.

Reply:
The latest is fine as well though.